.docker/app/nginx.conf

```diff
@@ -0,0 +1,33 @@
+user nginx;
+worker_processes auto;
+
+events {
+    worker_connections 4096;
+}
+
+http {
+    log_format combined_extra '$http_x_forwarded_for - $remote_user [$time_local] '
+                              '"$request" $status $bytes_sent '
+                              '"$http_referer" "$http_user_agent" '
+                              '$request_time';
+
+    access_log /dev/stdout combined_extra;
+    error_log /dev/stderr warn;
+
+    server {
+        listen      80;
+        server_name localhost _;
+        root        /app/public;
+
+        gzip on;
```

Comment:
What is gzip?

Reply:
gzip compression of the delivered items, cf http://nginx.org/en/docs/http/ngx_http_gzip_module.html

Reply:
It's an extension used by nginx to compress responses on-the-fly. It can help reduce traffic size by half in best cases :)